android/src/main/java/org/celo/rncdc/RNConfirmDeviceCredentialsPackage.java

```diff
@@ -1,5 +1,5 @@
 
-package com.reactlibrary;
+package org.celo.rncdc;
```

Comment:
Nit: How about `org.celo.devicecredentials`. The Java way is not to shorten the package names.